packages/contracts/contracts/voting/addresslist/AddresslistVoting.sol

```diff
@@ -185,37 +164,18 @@ contract AddresslistVoting is MajorityVotingBase {
         }
     }
 
-    /// @notice Checks if an account is on the address list at given block number.
-    /// @param _account The account address being checked.
-    /// @param _blockNumber The block number.
-    function isListed(address _account, uint256 _blockNumber) public view returns (bool) {
-        if (_blockNumber == 0) _blockNumber = getBlockNumber64() - 1;
-
-        return _addresslistCheckpoints[_account].getAtBlock(_blockNumber) == 1;
-    }
-
-    /// @notice Returns the length of the address list at a specific block number.
-    /// @param _blockNumber The specific block to get the count from.
-    /// @return The address list length at the specified block number.
-    function addresslistLength(uint256 _blockNumber) public view returns (uint256) {
-        if (_blockNumber == 0) {
-            _blockNumber = getBlockNumber64() - 1;
-        }
-        return _addresslistLengthCheckpoints.getAtBlock(_blockNumber);
-    }
-
     /// @inheritdoc MajorityVotingBase
-    function _canVote(uint256 _proposalId, address _voter) internal view override returns (bool) {
+    function _canVote(uint256 _proposalId, address _account) internal view override returns (bool) {
         Proposal storage proposal_ = proposals[_proposalId];
 
-        if (!_isVoteOpen(proposal_)) {
+        if (!_isProposalOpen(proposal_)) {
             // The proposal vote hasn't started or has already ended.
             return false;
-        } else if (!isListed(_voter, proposal_.parameters.snapshotBlock)) {
+        } else if (!isListedAtBlock(_account, proposal_.parameters.snapshotBlock)) {
```

Comment:
if we return in every `if` block there is no need to use `else if`. This would make from my point of view the code little bit more readable

Reply:
Fixed in ef2edbcfb60401b4c92da469d31320f523422a4c